Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+      description: The address group description.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.addresses
+      description: The address group addresses.
+      type: String
+    - contextPath: PrismaSase.AddressGroup.dynamic_filter
+      description: The address group filter.
+      type: String
+  - arguments:
+    - default: false
+      isArray: false
+      name: group_id
+      description: The name of the address group.
+      required: true
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    execution: false
+    description: Delete a specific address group.
+    name: prisma-sase-address-group-delete
+  - arguments:
+    - default: false
+      description: A specific url category to return. If not specified, all url categories will be returned.
+      isArray: false
+      name: id
+      required: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
+      isArray: false
+      name: folder
+      predefined:
+      - Shared
+      - Mobile Users
+      - Remote Networks
+      - Service Connections
+      - Mobile Users Container
+      - Mobile Users Explicit Proxy
+      required: false
+    - default: false
+      isArray: false
+      name: limit
+      description: The maximum number of results to return. Default is 50.
+    - default: false
+      isArray: false
+      name: page
+      description: Page number you would like to view. Each page contains page_size values. Must be used along with page_size.
+    - default: false
+      isArray: false
+      name: page_size
+      description: Number of results per page to display.
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    execution: false
+    description: Lists all custom URL categories.
+    name: prisma-sase-custom-url-category-list
+    outputs:
+    - contextPath: PrismaSase.CustomURLCategory.id
+      description: The URL category id.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.name
+      description: The URL category name.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.folder
+      description: The URL category folder.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.type
+      description: The URL category type.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.list
+      description: The URL category match list.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.description
+      description: The URL category description.
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASE folder location.
```
Is this the folder location or folder type/name?